test/commitments-specs.js

```diff
@@ -76,7 +76,7 @@ describe('commitments', () => {
 
   context('POST /commitments', () => {
     it('should create commitments with participant=participant and event=event', async () => {
-      let p1 = await models.db.participant.create({fbid: 'p1'})
+      await models.db.participant.create({fbid: 'p1'})
```

Comment:
Please still use the variable here instead of the literal (enables less churn when changes are made), you should be able to use `p1.fbid` below